beluga_amcl/docs/PARAMETERS.md

```diff
@@ -74,11 +76,100 @@ This page describes the parameters supported by Beluga AMCL including a comparis
+| `odom_alpha5` | Expected process noise in odometry's strafe estimate from translation for the omnidirectional drive model. | ✅ | ✅ |
+
+### Observation Model Parameters
+
+| Parameter | Description | Navigation AMCL   | Beluga AMCL |
+|-----------|-------------|:-----------------:|:-----------:|
+| `laser_model_type` | Which observation model to use. Supported models are `beam` and `likelihood_field` as described in [[3]](#3) with the same aggregation formula used in Nav AMCL. | ✅ | ✅ |
+| `laser_max_range` | Maximum scan range to be considered. See [compatibility notes](#compatibility-notes-ros1). | ✅ | ✅ |
+| `laser_min_range` | Minimum scan range to be considered. See [compatibility notes](#compatibility-notes-ros1). | ✅ | ✅ |
+| `laser_max_beams` | How many evenly spaced beams in each scan will be used when updating the filter. | ✅ | ✅ |
+| `laser_sigma_hit` | Standard deviation of the hit distribution used in likelihood field and beam models. | ✅ | ✅ |
+| `laser_z_hit` | Mixture weight for the probability of hitting an obstacle used in likelihood field and beam models. | ✅ | ✅ |
+| `laser_z_rand` | Mixture weight for the probability of getting random measurements used in likelihood field and beam models. | ✅ | ✅ |
+| `laser_z_max` | Mixture weight for the probability of getting max range measurements used in the beam model. | ✅ | ✅ |
+| `laser_z_short` | Mixture weight for the probability of getting short measurements used in the beam model. | ✅ | ✅ |
+| `laser_lambda_short` | Short readings' exponential distribution parameter used in the beam model. | ✅ | ✅ |
+| `laser_likelihood_max_dist` | Maximum distance to do obstacle inflation on map used in the likelihood field model. | ✅ | ✅ |
+| `do_beamskip` | Whether to ignore the beams for which the majority of the particles do not match the map in the likelihood field model. | ✅ | |
+| `beam_skip_distance` | Maximum distance to an obstacle to consider that a beam coincides with the map. | ✅ | |
+| `beam_skip_threshold` | Minimum percentage of particles for which a particular beam must match the map to not be skipped. | ✅ | |
+| `beam_skip_error_threshold` | Maximum percentage of skipped beams. Too many skipped beams trigger a full update to recover in case of bad convergence. | ✅ | |
+
+### Diagnostics Parameters
+
+| Parameter | Description | Navigation AMCL   | Beluga AMCL |
+|-----------|-------------|:-----------------:|:-----------:|
+| `std_warn_level_x` | Standard deviation upper bound for pose x position estimates before triggering a warning. | ✅ | ✅ |
+| `std_warn_level_y` | Standard deviation upper bound for pose y position estimates before triggering a warning. | ✅ | ✅ |
+| `std_warn_level_yaw` | Standard deviation upper bound for pose yaw rotation estimates before triggering a warning. | ✅ | ✅ |
+
+### Compatibility Notes {#compatibility-notes-ros1}
```

Comment:
Hmm, it doesn't look like this worked for a customer header ID.

Reply:
I'm OK with:
```suggestion
### Compatibility Notes ROS 1
```

Reply:
It appears both Github and VSCode add a `-{n}` suffix on duplicate anchor ids. I used that.